lesson19/src/js/index.js

```diff
@@ -0,0 +1,74 @@
+import { createAttributedElements } from './utiles/createAttributeWithEllement';
+import { showLoadingImage, removeLoading } from './modules/loading';
+import { addSliderContents } from './slider';
+import { addTabContents, changeTabs} from './news';
+
+function displayErrorMassage(error) {
+  const slideImageArea = document.getElementById("js-mainvisual-images-inner");
+  const textBox = createAttributedElements("p", "error-message");
+  textBox.textContent = error;
+  slideImageArea.appendChild(textBox);
+}
+
+const SLIDE_CONTENTS_URL = "https://api.json-generator.com/templates/9tm12BO1y5Xx/data?access_token=b0154huvd1stffra1six9olbgg34r4zofcqgwzfl";
+const NEWS_CONETNTS_URL = "https://api.json-generator.com/templates/2PqhEvPcqUZW/data?access_token=b0154huvd1stffra1six9olbgg34r4zofcqgwzfl";
```

Comment:
This is so clearly written!

Reply:
Thank you!!
This was also taught by Sho-san before!